Firmware/ultralcd.cpp

```diff
@@ -5730,6 +5714,25 @@ static void sheets_menu()
 
 void lcd_hw_setup_menu(void)                      // can not be "static"
 {
+    typedef struct
+    {// 2bytes total
+        int8_t status;
+        uint8_t experimental_menu_visibility;
```

Comment:
Would it make sense to use bool here? If so also the eeprom.h documentation should be updated